include/ArduinoJson/Print.hpp

```diff
@@ -7,6 +7,7 @@
 
 #pragma once
 
+#ifndef PARTICLE
```

Comment:
Just to be sure @foodbag, can you confirm that Particle doesn't define `ARDUINO`? because all other platforms do.